src/ResourceManager/ServiceFabric/AzureRM.ServiceFabric.psd1

```diff
@@ -75,13 +75,13 @@ FunctionsToExport = @()
 CmdletsToExport = 'Add-AzureRmServiceFabricApplicationCertificate', 
                'Add-AzureRmServiceFabricClientCertificate', 
                'Add-AzureRmServiceFabricClusterCertificate', 
-               'Add-AzureRmServiceFabricNode', 'Add-AzureRmServiceFabricNodeType', 
+               'Add-AzureRmServiceFabricNode', 
+               'Add-AzureRmServiceFabricNodeType', 
                'Get-AzureRmServiceFabricCluster', 
                'New-AzureRmServiceFabricCluster', 
                'Remove-AzureRmServiceFabricClientCertificate', 
                'Remove-AzureRmServiceFabricClusterCertificate', 
                'Remove-AzureRmServiceFabricNode', 
-               'Remove-AzureRmServiceFabricNodeType', 
```

Comment:
@QingChenmsft you cannot remove stuff once it's shipped?
Can you please explain what you mean by our production does not support this operation? Was it ever supported? If yes why it does not support it anymore without following proper deprecation process.